test/unit/black-button_test.js

```diff
@@ -174,6 +312,37 @@ describe('Black button testing', function() {
         });
     });
 
+    function sendUpdateLazyAttributes() {
+        var lazyAttributeUpdate = {
+            url: 'http://localhost:' + config.ngsi.server.port + '/v1/updateContext',
+            method: 'POST',
+            json: utils.readExampleFile('./test/unit/contextRequests/blackButtonSynchLazyRequest.json')
+        };
+
+        request(lazyAttributeUpdate, function(error, response, body) {
+            should.not.exist(error);
+            response.statusCode.should.equal(200);
+        });
+    }
+
+    function registerDevice(callback) {
+        var registerOptions = {
+            url: 'http://localhost:' + config.ngsi.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile('./test/unit/provision/synchronousButtonProvision.json'),
+            headers: {
+                'fiware-service': 'smartGondor',
```

Comment:
In blackbutton tests fiware-service will be blackbutton, and fiware-service-path things like telepizza, etc, aren't they?